locales/en.default.schema.json

```diff
@@ -1645,6 +1645,30 @@
         "price": {
           "name": "Price"
         },
+        "inventory": {
+          "name": "Inventory status",
+          "settings": {
+            "text_style": {
+              "label": "Text style",
+              "options__1": {
+                "label": "Body"
+              },
+              "options__2": {
+                "label": "Subtitle"
+              },
+              "options__3": {
+                "label": "Uppercase"
+              }
+            },
+            "inventory_threshold": {
+              "label": "Low inventory threshold",
+              "info": "Choose 0 to always show in stock if available."
+            },
+            "show_inventory_quantity": {
+              "label": "Show inventory count"
+            }
+          }
+        },
```

Comment:
Thanks Ludo, I've done it many times. I just prefer to do it when I'm sure we are not going to change anything in locales.

Reply:
Sounds good 👍 You can probably request them now since Yoann seemed to be ok we the copy and Monica and I haven't had new comments about it 🤔 They can take a few days to come back so to be safe, might as well do it now.